index.html

```diff
@@ -5,56 +5,66 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Sodiq's Portfolio</title>
-  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="mobile-style.css">
+  <link rel="stylesheet" media="screen and (min-width: 992px)" href="desktop-style.css">
```

Comment:
- [ ] As much as this is a valid way of including the `min-width`, it would be much better in terms of readability and maintainability if you added it to the CSS file. Kindly adjust. 🤗